packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/EnterAmount/Checkout/Payment/index.tsx

```diff
@@ -137,11 +166,32 @@ const Payment: React.FC<Props> = props => (
             ? renderCard(props.method)
             : renderFund(props.method)}
         </PaymentText>
+        <PaymentArrowContainer>
+          <Icon cursor name='arrow-right' size='20px' color='grey600' />
+        </PaymentArrowContainer>
+      </>
+    )}
+
+    {!props.method && (
+      <>
+        <SelectIconWrapper>
+          <Icon
+            cursor
+            name='plus-in-circle-filled'
+            size='22px'
+            color='blue600'
+            style={{ marginLeft: '4px' }}
+          />
+        </SelectIconWrapper>
+        <SelectPaymentText>
+          <FormattedMessage
+            id='modals.simplebuy.confirm.jump_to_payment'
+            defaultMessage='Select Cash or Card'
+          />
+        </SelectPaymentText>
+        <Icon cursor name='arrow-right' size='20px' color='grey600' />
```

Comment:
Just a small thing but it might be _even_ cleaner ✨ to move these to remove the `props.method` and move

the plus-in-circle-filled icon here: https://github.com/blockchain/blockchain-wallet-v4-frontend/pull/2536/files#diff-03c68ce533bdb95464ccc85ca7629558R145

this payment text here: https://github.com/blockchain/blockchain-wallet-v4-frontend/pull/2536/files#diff-03c68ce533bdb95464ccc85ca7629558R167

Just a suggestion 👍